crates/turborepo-lib/src/globwatcher/mod.rs

```diff
@@ -66,7 +66,7 @@ impl<T: Watcher> HashGlobWatcher<T> {
     /// Watches a given path, using the flush_folder as temporary storage to
     /// make sure that file events are handled in the appropriate order.
     #[tracing::instrument(skip(self, token))]
-    pub async fn watch(&self, token: StopToken) {
+    pub async fn watch(&self, token: StopToken) -> Result<(), ConfigError> {
```

Comment:
Do other callers need to handle this error as well?

Reply:
Good catch, the only caller happens in a macro that ended up suppressing the lint.